scenarios/writing_clamp_function_roc.yaml

```diff
@@ -1,97 +1,88 @@
 first: '1'
 screens:
   1: |
-    ┃
+    ┃🗌
   2: |
     clamp : 🗌
     clamp┃ = 🗌
   3: |
     clamp : 🗌
-    clamp = ┃
+    clamp = ┃🗌
   4: |
     clamp : 🗌 -> 🗌
-    clamp = \┃ ->
+    clamp = \┃🗌 ->
       🗌
   5: |
     clamp : 🗌 -> 🗌
     clamp = \v┃ ->
       🗌
   6: |
-      clamp : 🗌, 🗌 -> 🗌
-      clamp = \v, ┃ ->
-        🗌
+    clamp : 🗌, 🗌 -> 🗌
+    clamp = \v, ┃🗌 ->
+      🗌
   7: |
     clamp : 🗌, 🗌 -> 🗌
     clamp = \v, low┃ ->
       🗌
   8: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
-    clamp = \v, low, ┃ ->
+    clamp = \v, low, ┃🗌 ->
       🗌
   9: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
     clamp = \v, low, high┃ ->
       🗌
   10: |
     clamp : 🗌, 🗌, 🗌, 🗌 -> 🗌
-    clamp = \v, low, high, ┃ ->
+    clamp = \v, low, high, ┃🗌 ->
       🗌
   11: |
     clamp : 🗌, 🗌, 🗌 -> 🗌
     clamp = \v, low, high ->
-      ┃
+      ┃🗌
   12: |
-    clamp : a, *, * -> a
+    clamp : a, 🗌, 🗌 -> a
     clamp = \v, low, high ->
       v┃
   13: |
-    clamp : a, *, * -> a
```

Comment:
I think this step should be kept, the editor can't know `v` is not a function.
More importantly I think having a consistent UX will make the editor more pleasant to use (here the space key could mean go to the next node to edit).

Reply:
I see, the reason I removed this is because every AST should have a unique representation (including spacing) in text.
So I think we can add a Blank after v on space. I'll fix this tomorrow.